lib/screen/detail/preview/canvas_device_bloc.dart

```diff
@@ -272,6 +279,20 @@ class CanvasDeviceBloc extends AuBloc<CanvasDeviceEvent, CanvasDeviceState> {
       transformer: debounceSequential(const Duration(seconds: 5)),
     );
 
+    on<AppendDeviceEvent>((event, emit) async {
```

Comment:
What is the difference from `CanvasDeviceAppendDeviceEvent`?

Reply:
`CanvasDeviceAppendDeviceEvent ` is uncompleted and unused